Sunrise/de.lproj/Localizable.strings

```diff
@@ -175,4 +175,16 @@
 /* Get Directions */
 "Get Directions" = "Anfahrt";
 
+/* Discounts */
+"Discounts" = "Rabatt";
+
+/* Add discount code message */
+"Add your discount code bellow:" = "Fügen Sie Ihren Rabatt Code unten hinzu:";
+
+/* Discount */
+"Discount" = "Rabatt";
+
+/* No discounts */
+"No discounts - please redeem a code." = "Keine Rabatte - bitte erlöschen Sie einen Code.";
```

Comment:
just tried - sorry to say, I think you need to rethink the concept more generally. 

Try to put M0E20000000EAJR   in the cart ("Dress ana Alcazar multi" in size 34).  It has a cart discount applied that does not require a code.  So you see a Discount line but if you click the info icon it shows the error message. 

If I click the info icon next to a discount amount I want to see the name of the discount / how the discounts are mad up of multiple discount promotions and not an error that no code is applied.